### ISSUE TITLE
Test for Accessibility

### ISSUE THREAD

**Issue by digilou** [gwg-women/gwg-women-techmakers#81]
Perform checks for any accessibility issues.

**Comment by digilou** [comment]
@agonzalez0515 I'll run that one too! I always use WAVE and aXe.

**Comment by digilou** [comment]
Pushed updates to dev branch, but leaving this open for continued review up until the end. I'll stay vigilant in reviewing code to make sure my own updates aren't changed or new additions are WCAG 2.0 compliant.
Ideally, I'd like to add [Deque's react-axe testing module](https://github.com/dequelabs/react-axe), but I haven't been successful in getting it to work yet.

**Comment by tanyagupta** [comment]
@digilou Lighthouse is being actively used by Udacity/Google. That might be a good one too

**Comment by digilou** [comment]
@tanyagupta I was using the Chrome built-in and we tested at 97%. I can install the Node module, too, if others would find that helpful. Deque's module specifically targets React apps, and Lighthouse is fairly general to nail the basics within HTML like heading hierarchy, contrast, alt-text, etc.

**Comment by khusbuchandra** [comment]
We are doing so good in terms of accessibility. Thanks for all your effort @digilou . Really appreciate all the work done to get the app at its current level.

**Comment by digilou** [comment]
My pleasure @khusbuchandra!

**Comment by digilou** [comment]
@gwg-women/all-gwg-women I ran an accessibility automated audit with Lighthouse over our Heroku deployment: 100%.

Our syntax has held semantic. Color contrast is good. Everything that we've created on our end is keyboard accessible. Source reading order is good. Coordination between map and places is a little disorienting for me on VoiceOver, but a VO use may not find it so. Alt text has been given for images we have control over.

Good job, ladies! I'm satisfied with all the efforts towards accessibility within our project.
